schema-nft-marketplace.graphql

```diff
@@ -26,236 +26,239 @@ enum Network {
 }
 
 enum NftStandard {
-    ERC721
-    ERC721A
```

Comment:
I find that ERC721A is not part of openzepplin https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/token - should we add it here?